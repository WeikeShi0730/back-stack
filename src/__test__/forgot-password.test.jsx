import { render } from "@testing-library/react";
import ForgotPassword from "../components/forgot-password/forgot-password.component";

describe("<ForgotPassword />", () => {
  test("<ForgotPassword />", () => {
    const { baseElement } = render(<ForgotPassword />);
    expect(baseElement).toMatchSnapshot();
  });
});
