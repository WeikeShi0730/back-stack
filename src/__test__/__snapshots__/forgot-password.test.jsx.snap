// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForgotPassword /> <ForgotPassword /> 1`] = `
<body>
  <div>
    <div
      class="max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg m-auto mt-10 mb-4 bg-white rounded-lg border border-primaryBorder shadow-default py-8 px-10 text-sm md:text-base lg:text-lg"
    >
      <div
        class="text-center m-5"
      >
        Reset my password
      </div>
      <form>
        <div>
          <label
            class="text-sm md:text-base"
          >
            Email
          </label>
          <input
            class="w-full p-2 text-xs md:text-md text-primary border rounded-md outline-none transition duration-150 ease-in-out mb-4"
            id="email"
            name="email"
            placeholder="Your email"
            required=""
            type="email"
          />
        </div>
        <div
          class="flex justify-center items-center mt-6"
        >
          <button
            class="text-xs md:text-sm bg-gray-800 py-2 px-4 text-white rounded border focus:outline-none font-light"
            type="submit"
          >
            Send a password reset email
          </button>
        </div>
      </form>
    </div>
  </div>
</body>
`;
